okhttp-ws/src/main/java/com/squareup/okhttp/internal/ws/RealWebSocket.java

```diff
@@ -46,8 +47,9 @@
   private volatile boolean writerSentClose;
```

Comment:
Do multiple threads share a single client socket socket? My guess is that the UI will enqueue messages to be sent asynchronously, so a single write that fails shouldn't cause other writes to be treated as programming errors?